### ISSUE TITLE
Cannot configure input devices through hyprctl

### ISSUE THREAD

**Issue by REALERvolker1** [hyprwm/Hyprland#5195]
### Hyprland Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch HEAD at commit c5e28ebcfe00a510922779b2c568cfa52a317445  (props: bump ver 0.37.1).
Date: Sat Mar 16 14:51:49 2024
Tag: v0.37.1

flags: (if any)


System Information:
System name: Linux
Node name: joe
Release: 6.8.1-arch1-1
Version: #1 SMP PREEMPT_DYNAMIC Sat, 16 Mar 2024 17:15:35 +0000


GPU information: 
0000:00:02.0 VGA compatible controller [0300]: Intel Corporation Alder Lake-P GT1 [UHD Graphics] [8086:46a3] (rev 0c) (prog-if 00 [VGA controller])
0000:01:00.0 VGA compatible controller [0300]: NVIDIA Corporation GA106M [GeForce RTX 3060 Mobile / Max-Q] [10de:2520] (rev a1) (prog-if 00 [VGA controller])


os-release: NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://gitlab.archlinux.org/groups/archlinux/-/issues"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo


plugins:


```

</details>

### Bug or Regression?

Regression

### Description

There seems to be no way to configure individual input device settings via hyprctl after 1.37. Before the update, running the command `hyprctl keyword device:<device name>:enabled 0` would disable the device. Now, it does not. This is because there is no way to specify the device name anymore.

### How to reproduce

Run the command `hyprctl keyword device:<device name>:enabled 0`, with the device name taken from the output of `hyprctl devices`. On my machine, I would run `hyprctl keyword device:asup1205:00-093a:2003-touchpad:enabled 0` and I would get the error `config option <device:asup1205:00-093a:2003-touchpad:enabled> does not exist.`

### Crash reports, logs, images, videos

![image](https://github.com/hyprwm/Hyprland/assets/73304952/b4414de0-f8b5-4a45-b7b8-a7848b313fa9)

**Comment by sluedecke** [comment]
Same here, 1.37 breaks screen rotation which is handled by iio-hyprland.  iio-hyprland uses the syntax above to apply device transformations of input devices (touchpad, touchscreen).

**Comment by vaxerski** [comment]
#4856

**Comment by vaxerski** [comment]
tl;dr you can't `device:name:prop` anymore.

**Comment by tienbuigia** [comment]
in v0.37, I can use `hyprctl keyword device:name=<NAME>:enabled false` or getoption, but after updated to 0.37.1 I cannot do it anymore

**Comment by tienbuigia** [comment]
even when I try to create a variable in config file `$touchpadEnabled`, I cannot flip it using hyprctl keyword either or getoption.

[config & output](https://imgur.com/a/S5Pyt8V)

**Comment by REALERvolker1** [comment]
> tl;dr you can't `device:name:prop` anymore.

So what DO you run instead?

**Comment by REALERvolker1** [comment]
@vaxerski is there a new way to configure per-device settings through hyprctl?

**Comment by vaxerski** [comment]
```
$amongus = true
device {
    name = sus
   enabled = $amongus
}
```

```
hyprctl keyword '$amongus' false
```
(note the single quote so that bash doesnt parse it)

option 2 is putting the above snippet with your desired values in a .conf file and `source`ing it

**Comment by REALERvolker1** [comment]
This does not work. I have this section in my config file:
```
$touchpad_enabled = 1
device {
    name = asup1205:00-093a:2003-touchpad
    enabled = $touchpad_enabled
}
```

When I run `hyprctl keyword '$touchpad_enabled' 0`, it changes nothing. I have tried both integers and boolean types.

I do not want to enable automatic config reloading and use naive file sourcing, because this feature should ideally exist. Could you please add back a way to configure device settings, @vaxerski?

**Comment by vaxerski** [comment]
what's your hyprlang version? Strings with : were broken before like... 0.4.1 ish

**Comment by REALERvolker1** [comment]
```
 ~  pacman -Qi hyprlang
Name            : hyprlang
Version         : 0.5.0-1
```
0.5.0 from arch repos

**Comment by REALERvolker1** [comment]
It seems like arch got a bunch of hypr.* updates pushed out all at once, so that's why I'm bringing all this up now.

**Comment by vaxerski** [comment]
at any rate, changes to `keyword` to accept keys of categories is something to request over at hyprlang, not here

**Comment by kooskaspers** [comment]
Hmm this sucks a bit. I also used this logic to toggle my touchpad:

```
#toggle touchpad trackpad
bind=SUPER,T,exec,hyprctl keyword "device:bcm5974:enabled" false
bind=SUPERSHIFT,T,exec,hyprctl keyword "device:bcm5974:enabled" true
```

Let's see what https://github.com/hyprwm/hyprlang/issues/35 brings us.
